Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -239,8 +239,8 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
             clearButton.snp.makeConstraints { make in
                 make.height.centerY.equalToSuperview()
                 make.width.equalTo(40)
-                // Without this offset, the button moves 5 pixels when switching from UILabel mode to UITextField mode
-                make.right.equalToSuperview().offset(5)
+                // Without this offset, the button moves when switching from UILabel mode to UITextField mode
+                make.right.equalToSuperview().offset(5.5)
```

Comment:
wha??

Reply:
I know. This is our crazy widget overlaying that we have where the first tap creates a special fully selected overlay widget, and then the second tap hides that and enters a normal UITextField in the edit state. In the first state, we 'fake' a clear text button, so it has to align with the native one. The native one changes slightly as iOS updates. That was the value I needed to get this aligned :facepalm: